javascript/ql/src/Security/CWE-020/IncompleteUrlRegExp.ql

```diff
@@ -0,0 +1,68 @@
+/**
+ * @name Incomplete URL regular expression
```

Comment:
This name isn't descriptive enough.

Reply:
Changed to "Incomplete regular expression for hostnames", I have also changed the id and file name.